common/network-common/src/test/java/org/apache/spark/network/RequestTimeoutIntegrationSuite.java

```diff
@@ -121,7 +121,7 @@ public StreamManager getStreamManager() {
     // Second times out after 2 seconds, with slack. Must be IOException.
     TestCallback callback1 = new TestCallback();
     client.sendRpc(ByteBuffer.allocate(0), callback1);
-    callback1.latch.await(4, TimeUnit.SECONDS);
+    callback1.latch.await(60, TimeUnit.SECONDS);
```

Comment:
Does the commend `Second times out after 2 seconds, with slack. Must be IOException.` mean anything to the value that we put at here?